### ISSUE TITLE
Molecular Similarity

### ISSUE THREAD

**Issue by Leticia-maria** [caseykneale/OpenSMILES.jl#21]
Maybe, we can implement molecular similarity stuff, such like Tanimoto index

**Comment by caseykneale** [comment]
That one looks pretty easy to implement :). We could add it to this package short-term. If we can gather a few of these metrics it might be good to make a small package where others can contribute more too. Sort of as a companion, but abiding by the confederated design julia package developers like to follow. Do you wanna take a try at this?

**Comment by Leticia-maria** [comment]
> That one looks pretty easy to implement :). We could add it to this package short-term. If we can gather a few of these metrics it might be good to make a small package where others can contribute more too. Sort of as a companion, but abiding by the confederated design julia package developers like to follow. Do you wanna take a try at this?

Yes, absolutely, I will do it 🤗🤗 I was just waiting for your endorsement.

**Comment by timholy** [comment]
I still wonder why we don't all standardize on MolecularGraph? What's the advantage in having two packages that can't talk to one other implementing different functionality?
